src/style.css

```diff
@@ -0,0 +1,30 @@
+body {
+  margin: 0;
+}
+
+.stars--0,
+.stars--1,
+.stars--2,
+.stars--3,
+.stars--4,
+.stars--5 {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(images/star-active.svg);
+  background-position: 50%;
+  background-repeat: no-repeat;
+  height: 16px;
+  width: 16px;
+  padding-right: 4px;
+
```

Comment:
```suggestion
  padding-right: 4px;
```
don't forget about the rightmost stars 😉

<img width="149" alt="image" src="https://user-images.githubusercontent.com/55802630/173061730-ca10434a-5f8a-4a73-a49b-519753080aef.png">